client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1637,6 +1637,7 @@
   "messages.no-stock-available": "There is no stock available",
   "messages.no-transaction-other-facility": "Stock transactions are not recorded for vaccinations given in other facilities",
   "messages.no-vaccine-items-configured": "No vaccine items configured for this vaccine course. Stock transactions will not be recorded.",
+  "messages.error-printing-report": "Unable to print report",
```

Comment:
FYI there is an existing translation key of 
"error.failed-to-generate-report" 
in our common.json

Maybe could use this one instead? It more or less means the same thing to me, and already has existing translations in other languages. Over to you :)

Reply:
Also I'm just looking and seeing that we are not propagating any error types from the back end to front end, and handling them selectively (you can see that in the api for generateReport, we either get the report node, OR throw a generic error).  So we aren't getting any information about the error as a user.

I'm wondering if it would be better to call this 'error' when we give a toast rather than 'unable' because unable makes me wonder if it is a permission denied thing rather than an issue with the back end.